index.rst

```diff
@@ -154,38 +154,7 @@ Core developers and contributors alike will find the following guides useful:
 * `How to Contribute to Open Source <https://opensource.guide/how-to-contribute/>`_
 * `Building Welcoming Communities <https://opensource.guide/building-community/>`_
 
-Guide for contributing to Python:
-
-+------------------------+---------------------+-----------------------+---------------------+
-| New Contributors       | Documentarians      | Triagers              | Core Developers     |
-+========================+=====================+=======================+=====================+
-| :doc:`setup`           | :doc:`docquality`   | :doc:`tracker`        | :doc:`coredev`      |
-+------------------------+---------------------+-----------------------+---------------------+
-| :doc:`help`            | :doc:`documenting`  | :doc:`triaging`       | :doc:`developers`   |
-+------------------------+---------------------+-----------------------+---------------------+
-| :doc:`pullrequest`     | :ref:`style-guide`  | :ref:`helptriage`     | :doc:`committing`   |
-+------------------------+---------------------+-----------------------+---------------------+
-| :doc:`runtests`        | :ref:`rst-primer`   | :doc:`experts`        | :doc:`devcycle`     |
-+------------------------+---------------------+-----------------------+---------------------+
-| :doc:`fixingissues`    | :ref:`translating`  |                       | :doc:`motivations`  |
-+------------------------+---------------------+-----------------------+---------------------+
-| :doc:`communication`   |                     |                       | :ref:`office hour`  |
-+------------------------+---------------------+-----------------------+---------------------+
-| :doc:`gitbootcamp`     |                     |                       |                     |
-+------------------------+---------------------+-----------------------+---------------------+
```

Comment:
This table grouped the page by intended audience, which is somewhat different from the grouping implemented in the sidebar.  I think it might be better to keep it and possibly update it.
Here it would also be a good place to list the "recommended readings" that were suggested in the appendix.

Reply:
That could make sense, and is essentially the approach I was planning for a followup PR if the rest of the restructuring wasn't done here. One idea: Instead of listing every individual page, which is way longer and a lot more work to maintain and risk of desync, we could just list the category index pages if most or all of the pages in the category apply.